src/eeprom.c

```diff
@@ -0,0 +1,82 @@
+#include "eeprom.h"
+
+#define BYTES_PER_WORD 4
+
+/**
+ * @brief Function to read eeprom data from a specific address.
+ *
+ * @param[in] start_address Address to read from.
+ * @param[in] num_elements Number of bytes to be read.
+ * @param[out] output_data Pointer to a word to save the read data.
```

Comment:
- EEPROM all caps.
- `output_data` says pointer to a word, but it is an `uint8_t`. Maybe we should call it buffer? (we actually pretend to be able to read multiple elements)

Reply:
* Fixed
* you are right, updated with buffer